doc: Documentation about Wi-Fi based LwM2M location assistance (#9740)
nrfconnect/sdk-nrf: doc/nrf/libraries/networking/lwm2m_location_assistance.rst

To send the location request for the cell-based location, call the :c:func:`location_assistance_ground_fix_request_send` function.

Wi-Fi based location
====================

Wi-Fi based location uses the nearby Wi-Fi access points for estimating location.
The library uses a collection of Visible Wi-Fi Access Point objects (ID 33627) for storing the information about nearby Wi-Fi access points.

> To populate the objects, first enable the Kconfig option for the access point scanner :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_WIFI_AP_SCANNER` and call the :c:func:`lwm2m_wifi_request_scan` to request the access point scan.

Reviewer: ```suggestion
To populate the objects, first enable the Kconfig option for the access point scanner :kconfig:option:`CONFIG_LWM2M_CLIENT_UTILS_WIFI_AP_SCANNER` and call the :c:func:`lwm2m_wifi_request_scan` function to request the access point scan.
```

> The Ground Fix Location object is used similar to the cell-based location when sending the location request.

Reviewer: ```suggestion
The Ground Fix Location object is used in the same manner as the cell-based location when sending the location request.
```
